cmd/marker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM centos:centos7
+COPY marker /marker
+COPY .version /.version
+ENTRYPOINT [ "./marker"]
```

Comment:
I think you can just `/marker` remove the `.`

Reply:
Looks like just "marker" works too.

Done